api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -110,6 +114,20 @@ public void setUserProvider(Provider<User> userProvider) {
       fcCluster = this.notebooksService.createCluster(
           project, NotebooksService.DEFAULT_CLUSTER_NAME, userProvider.get().getEmail());
     }
+    if (fcCluster.getStatus() != null && fcCluster.getStatus().equals(org.pmiops.workbench.notebooks.model.ClusterStatus.ERROR)) {
```

Comment:
Can avoid the null check by reordering this: `ClusterStatus.ERROR.equals(fcCluster.getStatus());`, here and below